__tests__/healthcareProfessional.test.ts

```diff
@@ -81,6 +83,494 @@ describe('createHealthcareProfessional', () => {
         expect(error.errorCode).toBe(ErrorCode.CREATEPROFFESIONAL_FACILITYIDS_REQUIRED)
         expect(error.httpStatus).toBe(400)
     })
+
+    test('failing: throws an error if locale and characters used don\'t match', async () => {
+        // I iterate through an array of locales to accommodate future language additions.
+        const localeToBeTested = [gqlTypes.Locale.JaJp, gqlTypes.Locale.EnUs]
+
+        for (const locale of localeToBeTested) {
+            const createFailingHealthcareProfessional = {
+                query: createHealthcareProfessionalMutation,
+                variables: {
+                    input: {
+                        facilityIds: sharedFacilityIds,
+                        names: [testData.generateFailingNames('namesContainsInvalidAlphabet', locale)],
+                        degrees: [testData.generateDegreeInput()],
+                        specialties: [testData.generateSpecialty()],
+                        spokenLanguages: testData.generateSpokenLanguages(),
+                        acceptedInsurance: [testData.generateAcceptedInsurance()]
+                    }
+                }
+            } as gqlMutation<CreateHealthcareProfessionalInput>
+    
+            const createProfessionalResult = await request(gqlApiUrl).post('/').send(createFailingHealthcareProfessional)
+            const createdProfessional
+                         = createProfessionalResult.body.data.createHealthcareProfessional as HealthcareProfessional
+                
+            expect(createdProfessional).toBeFalsy()
+            expect(createProfessionalResult.body?.errors).toBeDefined()
+            expect(createProfessionalResult.body?.errors[0].extensions.errors[0]).toBeDefined()
+            expect(createProfessionalResult.body?.errors[0].extensions.errors.length).toBeGreaterThanOrEqual(3)
+    
+            const errorFirstName = createProfessionalResult.body?.errors[0].extensions.errors[0] as Error
+            const errorlMiddleName = createProfessionalResult.body?.errors[0].extensions.errors[1] as Error
+            const errorlLastName = createProfessionalResult.body?.errors[0].extensions.errors[2] as Error
+    
+            expect(errorFirstName.field).toBe('names[0].firstName')
+            expect(errorFirstName.errorCode).toBeDefined()
```

Comment:
Testing the Errorcode for this test was a bit difficult, since depending on the locale the Errorcode changes (how it is implemented now)